capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] You are required to return a capitalized word (eg. 'Maximilianus'), not an uppercase (eg 'MAXIMILIANUS'). Kindly fix that.